dags/utils/django.py

```diff
@@ -33,35 +18,44 @@
 
 
 def django_add_to_sys_path() -> None:
-    """Fonction qui ajoute la racine de notre projet Django à sys.path"""
-    # On ajoute le dossier de 2 niveaux au dessus de celui du script
-    # pour pouvoir importer le settings.py
-    # Arborecence actuelle:
-    # - core/
-    # - dags/utils/django.py
-
-    # Premier des 3x .parent sert à récupérer le dossier,
-    # les 2 autres à remonter à la source de notre repo
-    path_2_levels_up = str(Path(__file__).resolve().parent.parent.parent)
-    sys.path.insert(0, path_2_levels_up)
-
-
-def django_setup_full() -> None:
-    """Initialisation complète de l'environement Django pour pouvoir,
-    entre autres, importer et utiliser les modèles dans Airflow.
+    """Adds Django project root to sys.path, based on
+    current directory structure:
+     - core/ = Django root
+     - dags/utils/django.py"""
+    django_root = str(Path(__file__).resolve().parent.parent.parent)
+    sys.path.insert(0, django_root)
+
+
+def django_settings_to_dict(settings) -> dict:
+    """Returns useful information from settings
+    as a JSON-compatible dict to help us debug
+    (e.g. when setting up e2e Airflow tests)"""
```

Comment:
En vrai, ici, tu retourne les info de la connexion à la DB uniquement

```suggestion
def get_django_db_connexion_settings(settings) -> dict:
    """
    Returns database connection informations from Django settings as a dict to help us debug
    (e.g. when setting up e2e Airflow tests)
    """
```

Reply:
Oui, mais c'est une abstraction pour avoir à renommer en permanence la fonction au fur et à mesure qu'on lui ajoute des infos